### PR TITLE
Added section on how to access pnld website

### PR DESCRIPTION
**Pull request by joe-fol**
Screenshot 2024-05-28 at 13 08 00

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The offence code data and organisation unit data are the output of a build proce
 The output offence code data (`output-data/data/offence-code.json`) is generated by combining a number of different data sources into one set of offence codes. Input data is:
 
 - **b7-overrides**. Any offence code that is referenced in this file must have its offence category set to "B7" so that it is ignored by Bichard. NB: This exists for compatibility with the legacy dataset and should be removed in future.
-- **cjs-offences**. Offence code data exported from the data standards team and published [here(]https://www.gov.uk/guidance/criminal-justice-system-data-standards-forum-guidance#data-standards-catalogue)
+- **cjs-offences**. Offence code data exported from the data standards team and published [here](https://www.gov.uk/guidance/criminal-justice-system-data-standards-forum-guidance#data-standards-catalogue)
 - **pnc-ccjs-cjs-offences**. Data exported from the PNC
 - **pnld-offences** Data exported from the [PNLD](https://www.pnld.co.uk)
```
Comment (chubberlisk) [inline README.md:39]:
```suggestion
- **pnld-offences** Data exported from the [PNLD (Police National Legal Database)](https://www.pnld.co.uk)
```

I didn't know what PNLD was and had to look it up, so just adding the acronym is useful.

**Review by Senny10 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -46,8 +46,30 @@ $ npm run download-offence-code-data
 
 # Combine all sources into the final output in output-data/
 $ npm run merge-offence-data
+
+```
+###  Accessing the PNLD website
+
+We use puppeteer to interact with the PNLD in the browser. From time to time we experience differences in the browser through updates to html or broken links so its helpful debugging. To access it in the browser go to [PNLD](https://www.pnld.co.uk/standard-offence-wording-extracts/) and look in the table bellow to find out where to get the credentials
```
Comment (chubberlisk) [inline README.md:53]:
```suggestion
We use [Puppeteer](https://pptr.dev/) to interact with the PNLD in the browser. From time to time we experience differences in the browser through updates to HTML or broken links so it's helpful debugging.

To access it in the browser go to [PNLD](https://www.pnld.co.uk/standard-offence-wording-extracts/) and look in the table below to find out where to get the credentials.
```

Just to help for readability!

File: README.md [hunk 3]
```diff
@@ -46,8 +46,30 @@ $ npm run download-offence-code-data
...
+
+The function `PnldFileDownloader` is called by the script `download-offence-code-data.ts` and needs four environment variables in order to access the PNLD service.
+
+| environment variable    | description        |
+| ----------------------- | ----------------------------------------------------------- |
+| PNLD_USERNAME           | User name can be found in 1password in the shared vault     |
+| PNLD_PASSWORD           | Password can be found in 1password in the shared vault      |
+| PNLD_LOGIN_URL          | https://www.pnld.co.uk/standard-offence-wording-extracts/   |
+| PNLD_DOWNLOAD_URL       | This depends on the zip file we want to download: <br> Full extract: https://www.pnld.co.uk/standard-offence-wording-extracts/full-extract <br>Monthly Update (Current Month): https://www.pnld.co.uk/standard-offence-wording-extracts/monthly-delta-extract<br>Monthly Updates (Last Month): https://www.pnld.co.uk/standard-offence-wording-extracts/monthly-delta-extract-1-month-prior), [Monthly Updates (2 Months Ago): https://www.pnld.co.uk/standard-offence-wording-extracts/monthly-delta-extract-2-months-prior
```
Comment (chubberlisk) [inline README.md:62]: Is it possible to make the types of zip files a bulleted list within the table description? 🤔

I think we have an stray `[` btw on "Monthly Updates(2 months ago)!

**Review by chubberlisk — APPROVED** [review]
Just some readability and formatting suggestions, all good otherwise - feel free to merge afterwards!

**Pull request merged**